javascript/index.js

```diff
@@ -37,28 +42,79 @@ const displayCurrentWeatherSummary = (response) => {
   $("#current-summary").html('');
   $('#current-summary').append(`
     <div class="summary-left">
-      <h5>
-        <span class="currently-location">${response.data.id}</span>
-      </h5>
-      <h5>
-        <span id="currently-summary">${response.data.attributes.current_forecast.summary}</span>
-      </h5>
-      <h5>
-        <span class="currently-time">${response.data.attributes.current_forecast.time}</span>
-      </h5>
-      <h5>
-        <span id="currently-temperature">Now ${response.data.attributes.current_forecast.temp}</span>&deg;
-      </h5>
-      <h5>
-        <span id="currently-apparent-temperature">Feels Like ${response.data.attributes.current_forecast.feels_like}</span>&deg;
-      </h5>
-      <h5>
-        <span id="currently-humidity">Humdiity ${response.data.attributes.current_forecast.humidity}%</span>
-      </h5>
-      <h5>
-        <span id="currently-uvIndex">UV Index ${response.data.attributes.current_forecast.uv_index}</span>
-      </h5>
+      <h5><span class="currently-location">${response.data.id}</span></h5>
+      <h5><span id="currently-summary">${response.data.attributes.current_forecast.summary}</span></h5>
+      <h5><span class="currently-time">${response.data.attributes.current_forecast.time}</span></h5>
+      <h5><span id="currently-temperature">Now ${response.data.attributes.current_forecast.temp}</span>&deg;</h5>
+      <h5><span id="currently-apparent-temperature">Feels Like ${response.data.attributes.current_forecast.feels_like}</span>&deg;</h5>
+      <h5><span id="currently-humidity">Humdiity ${response.data.attributes.current_forecast.humidity}%</span></h5>
+      <h5><span id="currently-uvIndex">UV Index ${response.data.attributes.current_forecast.uv_index}</span></h5>
     </div>
   `);
   $('#current-summary').css('display', 'inherit');
 }
+
+const getFavorites = () => {
+  let url = `${productionUrl}/api/v1/favorites?api_key=${api_key}`
+  fetch(url)
+  .then((response) => response.json())
+  .then((res) => {
+    displayFavorites(res);
+  })
+  .catch(error => {
+    console.log(error)
+    });
+}
+
+const displayFavorites = (response) => {
+  $("#favorites").html('');
+  
+  let favorites = response.data
+  
+  favorites.forEach(function(location) {
+    $('#favorites').append(`
+    <h3><span id="favorite">${location.meta.data.id}</span></h3>
```

Comment:
Indentation here.
Also to make this a link you can just wrap in an `<a></a>` tag instead of a span. Also, I would make the `id` for the a tag the `location.meta.data.id`. And inside the a tag your could have the location name(I'm not sure what attribute that is, in your json response).

`<h3><a id="${location.meta.data.id}">"${whatever the location name is in the json}"</a></h3>`

Reply:
Perfect! I'm still learning how to make the most of CSS and targeting.  Implemented!